packages/coordinator/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import DatePicker, { DatePickerProps } from "./DatePicker";
+import { Meta } from "@storybook/react";
+
+export default {
+  component: DatePicker,
+  title: "Components/Date Picker",
+  parameters: { layout: "padded" },
+} as Meta;
+
+const props: DatePickerProps = {
+  label: "בחר שעה",
+  onChange: () => {},
+  value: new Date(),
```

Comment:
Please set a curated date, otherwise the snapshot will fail on every run

Reply:
done